src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
>Since config file is user editable it is a really good idea to store enumerations by name (as strings) as @evsh introduced in one of his last PRs

just got it working, but the new declared enum must accompany with a `Q_ENUM_NS(...)` macro which is only available on `Qt >= 5.8`.  Also must add `Q_NAMESPACE` to `namespace BitTorrent` to support it.
Any comment?